src/doc/trpl/looping.md

```diff
@@ -54,7 +54,7 @@ The other kind of looping construct in Rust is the `while` loop. It looks like
 this:
 
 ```{rust}
-let mut x = 5u;       // mut x: uint
+let mut x = 5u32;       // mut x: u32
```

Comment:
If no suffix is preferred in this case, I'd be happy to make that change instead, and create a new PR.  I was mostly interested in clearing compiler warnings due to the deprecated uint type.